tasks/variables.js

```diff
@@ -1,12 +1,12 @@
 module.exports = function( grunt ) {
 	grunt.registerTask( "buildVariables", function() {
-		var jsass = require( "jsass-vars" );
-		var path = require( "path" );
-		var varPath = path.join( process.cwd(), "scss/variables/*" );
+		var jsass = require( "jsass-vars" ),
```

Comment:
One var is no longer a requirment but we need to update our version of JSCS for it to not complain.